vite.config.js

```diff
@@ -63,10 +64,18 @@ export default defineConfig(({ mode, packageType }) => {
       }),
     );
   }
+  const allow = [searchForWorkspaceRoot(process.cwd())];
+  const matrixSdkCryptoWasmPath =
+    "node_modules/matrix-js-sdk/node_modules/@matrix-org/matrix-sdk-crypto-wasm";
```

Comment:
We should additionally check for the existence of `node_modules/@matrix-org/matrix-sdk-crypto-wasm` to support cases where Yarn decides to hoist the package out of matrix-js-sdk (i.e. when you're not using a linked matrix-js-sdk, but simply a linked matrix-sdk-crypto-wasm)

Reply:
I tried that and this works automatically
if i do not link it can load the rust crypto without any yarn adjustments

Reply:
oh but if that is linked. got it
I had it like that before but reverted because i did not see the case where you only want to link crypto but the js-sdk

Reply:
Yep, I recall Valere originally opened this PR because of that use case